extension.js

```diff
@@ -219,6 +219,12 @@ function activate( context )
         } );
     }
 
+    function buildGlobs(includeGlobs, excludeGlobs) {
+        return []
+            .concat(includeGlobs)
+            .concat(excludeGlobs.map(g => `!${g}`));
```

Comment:
Adding the `!` to make this an exclusion rule. Wasn't sure if we need to wrap with parentheses too.